src/nunit.analyzers/UseAssertMultiple/UseAssertMultipleCodeFix.cs

```diff
@@ -113,8 +129,21 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
                             SyntaxFactory.Argument(parenthesizedLambdaExpression)
                         })))).WithAdditionalAnnotations(Formatter.Annotation);
 
+            if (endOfLineTrivia is not null)
+            {
+                // Add the remembered blank line.
+                assertMultiple = assertMultiple.WithLeadingTrivia(endOfLineTrivia.Value);
+            }
+
+            // Add new line after the Assert.Multiple statement.
+            assertMultiple = assertMultiple.WithTrailingTrivia(SyntaxFactory.CarriageReturnLineFeed);
```

Comment:
Will this also work for codebases that use LF or do we need to either extract the current setting or use ElasticCarriageReturnLineFeed (I tried to read and understand dotnet/roslyn#202, dotnet/roslyn#19306, but I ended up more confused).

Reply:
You an me both. I initially was looking for `Environment.NewLine` but that raised an issue that analyzers are not allowed to depend on environment variables. 

Using `ElasticCarriageReturnLineFeed` they were [removed](https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntaxfactory.elasticcarriagereturnlinefeed?view=roslyn-dotnet-4.3.0): (**emphasis** mine)

> Elastic trivia are used to denote trivia that was not produced by parsing source text, and **are usually not preserved during formatting**."

According to a [comment](https://github.com/dotnet/roslyn/issues/19306#issuecomment-920139354) to the issue you referenced: (**emphasis** mine)

> Note that Roslyn implements a formatting pass after the application of a code fix or refactoring. During this pass, elastic nodes and **nodes marked with the formatter annotation** will be updated to use the correct line endings.

The `assertMultiple` statement is annotated with the `Formatter.Annotation`.

This seems to be correct as when updating the code to use `LineFeed`  the test still work and introduce CRLF.

Updated the code to add the CRLF before mentioning the Formatter to make it clearer.